### PR TITLE
Encryption at rest fix

### PR DESCRIPTION
**Pull request by marinsalinas**

### CODE REVIEW

**Review by PacoDw — APPROVED** [review]
Nice 👍 !

File: mongodbatlas/resource_mongodbatlas_encryption_at_rest.go [hunk 1]
```diff
@@ -200,3 +199,28 @@ func resourceMongoDBAtlasEncryptionAtRestDelete(d *schema.ResourceData, meta int
...
 }
+
+func resourceMongoDBAtlasEncryptionAtRestUpdate(d *schema.ResourceData, meta interface{}) error {
+	conn := meta.(*matlas.Client)
+	projectID := d.Id()
+
+	encrypt, _, err := conn.EncryptionsAtRest.Get(context.Background(), projectID)
+
+	if err != nil {
+		return fmt.Errorf("error getting enctryption at rest information: %s", err)
```
Comment (themantissa) [inline mongodbatlas/resource_mongodbatlas_encryption_at_rest.go]: typo =  encryption.

**Review by themantissa — COMMENTED** [review]
LGTM for AWS minus one typo in error I've commented on.   However we'll need the same update, I believe, for all 3 cloud providers.   Edit: GCP, Azure will be corrected in separate PR.

**Review by themantissa — COMMENTED** [review]
LGTM - appreciate the add of the website fix.

**Review by themantissa — APPROVED** [review]

**Pull request merged**